h2o-algos/src/test/java/hex/tree/DHistogramTest.java

```diff
@@ -190,14 +190,14 @@ public void testUpdateHistoWithIntOpt() {
             SharedTreeModel.SharedTreeParameters.HistogramType.UniformAdaptive, 42L, null, null, false);
     histoOpt.init();
 
-    histoOpt.updateHisto(weights, null, dataInt, ys, null, rows, N, 0);
+    histoOpt.updateHisto(weights, null, dataInt, ys, null, rows, N, 0, null);
```

Comment:
I think it would be cleaner to create second `updateHisto` method that would call the one extended with the last parameter `null` and all original usage would remain unchanged

Reply:
Good point, will change.